site/cypress/e2e/Cypress-Feature/polls.spec.js

```diff
@@ -204,6 +204,11 @@ describe('Test cases revolving around polls functionality', () => {
         cy.get('#enable-timer').check();
         cy.get('#timer-inputs').should('be.visible');
         cy.get('#enable-timer').should('be.checked');
+        // Testing 'Show Answers' disappearing after poll becomes survey
+        cy.get('#show-answer-release').should('be.visible');
```

Comment:
Since these are new elements being added, can you add "data-testids" to the HTML elements, and use them as selectors here? Ideally this entire test would be refactored to use data-testids, but that can be a future PR.